data/json/items/gun/shot.json

```diff
@@ -197,9 +221,10 @@
   },
   {
     "id": "saiga_12",
-    "copy-from": "shotgun_base",
+    "copy-from": "shotgun_magazine",
```

Comment:
You only use this class once. It therefore doesn't reduce any redundancy and just obfuscates what's going on